### ISSUE TITLE
Base button styling doesn't meet WCAG 2.1AA contrast ratio

### ISSUE THREAD

**Issue by dantrewartha** [BUNN-Digital/design-system#28]
Current base button style is `--color-gray-600` (`#777777`) text on a background of `--color-gray300` (`#e0e0e0`). This gives a contrast ratio of 3.39:1 which does not meet the WCAG AA standard for normal text, or the AAA standard for normal or large text. 

Foreground text should be changed to a darker value like `--color-gray-700` (#444444) to meet AA and AAA standards for all text sizes with a contast ratio of 7.37:1
